### PR TITLE
Old login redesign pre-styling clean up

### PR DESCRIPTION
**Pull request by goran-peoski-work**
Mostly moving the old components into a single place that will be easy to remove later on after new Login flow is created.

There is slight addition / transfer of the `logout()` function so that attendee can do `SPARK.logout()` in their browser console.

### CODE REVIEW

File: src/core/spark.ts [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+  BUILD_PULL_REQUESTS,
+  BUILD_REPOSITORY_URL,
+  BUILD_SHA1,
+  BUILD_TAG,
+} from "secrets";
+
+import { FIREBASE } from "./firebase";
+
+const logout = () =>
+  void FIREBASE.auth.signOut().then(console.log).catch(console.error);
```
Comment (alexxpetrov) [inline src/core/spark.ts]: weird console usages if you ask me

**Review by alexxpetrov — APPROVED** [review]
LGTM, note the console tho

File: src/core/spark.ts (hunk 1 above)
Comment (goran-peoski-work) [reply src/core/spark.ts]: Yeah, I meant to replace it with more elaborate one, like the capture of error

Comment (goran-peoski-work) [reply src/core/spark.ts]: Done 4425bc5465061b654e0802f90fd88d3e4d895b70

**Pull request merged**